source/plugin/data/serialization.rst

```diff
@@ -123,3 +125,56 @@ from persistent files using the :doc:`Configurate Library <../configuration/inde
         final DataView container = translator.translate(node);
         return deserializeHealth(container);
     }
+
+
+DataFormat
+==========
+An alternative to DataTranslators is to use :javadoc:DataFormat, which allows you to store a ``DataContainer`` as a HOCON, JSON or NBT file. You can also retrieve data using ``DataFormat``.
```

Comment:
It doesn't necessarily create files. It transforms ``DataContainer``s into HOCON, JSON or NBT _formats_, which _could_ be converted into a file, but in your example below, you don't create a file. 

That last sentence should read something like:

> You can also re-create ``DataContainer``s using these ``DataFormat``s.